PHYSICS/AddRope.md

```diff
@@ -35,4 +85,51 @@ Rope will sometimes contract and fall to the ground like you'd expect it to, but
 * **unkPtr**: Unknown pointer, always 0 in original scrips.
 
 ## Return value
-A script handle for the rope
\ No newline at end of file
+A script handle for the rope
+
+## Examples
+```lua
+RegisterCommand("new_rope", function(source, args, rawCommand)
+
+    Citizen.CreateThread(function()
+        -- Get the handle for the player's ped
+        local pedHandle = PlayerPedId()
+
+        -- Ensure that the rope textures are loaded
+        while not RopeAreTexturesLoaded() do
+            RopeLoadTextures()
+            Citizen.Wait(0)
+        end
+
+        if not RopeAreTexturesLoaded() then

```

Comment:
This seems unneeded, there is no way to get here.

Reply:
Yeah, totally right. I forgot about this part after I made the last change to the while loop, thanks for catching it!